### ISSUE TITLE
Is tcDontShowErrors effective?

### ISSUE THREAD

**Issue by eselje** [fdbozzo/foxbin2prg#33]
I'm calling FoxBin2Prg on DBFs like 
`DO C:\DEV\FOX\FOXBIN2PRG\FOXBIN2PRG.PRG WITH <tableName>, "BIN2PRG", "D", .T., .T., .T., .T.`

I want it to be completely silent. No progress, no errors. Despite this I still get a messageBox when there's an error letting me know there's an error, and it attempts to open the error log in Notepad. What am I doing wrong?

**Comment by fdbozzo** [comment]
Hi Eric!

You are not using the parameters correctly.
In the wiki (https://github.com/fdbozzo/foxbin2prg/wiki), at the bottom, there is a table of parameters and there value types.

Almost all parameters are character-type

Those parameters are not boolean, and they are all described in the wiki.

Best Regards!

**Comment by eselje** [comment]
Ok got it working very well, thank you. 

`DO C:\DEV\FOX\FOXBIN2PRG\FOXBIN2PRG.PRG WITH "%cTable", "D", '', .F., '1', '0', '1'`

I did tweak my copy of FoxBin2Prg because I was making subsequent calls that were erroring out quickly (not a problem after I tweaked the parameters):

`THIS.c_LogFile			= ADDBS( THIS.c_TempDir ) + 'FoxBin2Prg_Debug' +TTOC(DATETIME(),1)+TRANSFORM(SECONDS())+'.LOG'
THIS.c_ErrorLogFile		= ADDBS( THIS.c_TempDir ) + 'FoxBin2Prg_Error' +TTOC(DATETIME(),1)+TRANSFORM(SECONDS())+'.LOG'
`

**Comment by fdbozzo** [comment]
Hi Eric,

If you want the fastest working of foxbin2prg, you should use the exe and not the prg, and use it in object mode, not as standalone program.

That way you can customize every aspect you need.

It's documented in the Wiki, but you have also the scripts I've made (*.vbs) for example.

**Comment by eselje** [comment]
I switched it to object mode and it cut the time from 19.779 down to 8.41
seconds. That's impressive! I did seem to have to put a FoxBin2Prg.cfg file
into my working folder for it to work in object mode though.

I'll share a blog post w/ you about what I'm doing here, but it's
essentially comparing a two folders worth of DBFs to see the schema
differences. There's no DBC. It collects all of the results into two text
files and sends them to Beyond Compare for an easy visualization of the
differences between the tables in the two folders.

Thanks!

Eric
